components/post_view/commented_on/commented_on.jsx

```diff
@@ -74,10 +69,9 @@ export default class CommentedOn extends PureComponent {
                 <span>
                     <FormattedMessage
                         id='post_body.commentedOn'
-                        defaultMessage='Commented on {name}{apostrophe} message: '
+                        defaultMessage='Commented on {name} message: '
```

Comment:
Does not match with the en.json.

Reply:
Ah right. I missed the `{name}'s message` the first time as well